templates/index.html

```diff
@@ -2,17 +2,97 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+    <div class="jumbotron">
+        <h1 class="display-4">To-Do App</h1>
+        <p class="lead">Just another to-do app.</p>
+    </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <div class="list-group mb-4 container">
+                {% if items is defined and items %}
+                    {% for item in items %}
+                        <div class="list-group-item d-flex justify-content-between align-items-center row">
+                            <div class="col-8">{{ item.title }}</div>
+                            <div class="col-2">
+                                {% if item.status == 'Not Started' %}
+                                    <form id="complete_item" method="post" action="">
+                                        <a href="#">
+                                    <span class="badge badge-danger badge-pill red"
+                                          onclick="complete_item({{ item.id }})">{{ item.status }}</span>
+                                        </a>
+                                    </form>
+                                {% elif item.status != 'Not Started' %}
+                                    <span class="badge badge-success badge-pill red">{{ item.status }}</span>
+                                {% endif %}
+                            </div>
+                            <div class="col-2">
+                                <form id="delete_item" method="post" action="">
+                                    <a href="#">
```

Comment:
You could achieve the same effect using a `<button>` element, which would avoid the need to use javascript in the below.